flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTableSourceSinkFactoryTestBase.java

```diff
@@ -410,24 +414,64 @@ public void testTableSinkWithLegacyProperties() {
     }
 
     protected Map<String, String> createKafkaSinkProperties() {
-        return new TestTableDescriptor(
-                        new Kafka()
-                                .version(getKafkaVersion())
-                                .topic(TOPIC)
-                                .properties(KAFKA_PROPERTIES)
-                                .sinkPartitionerFixed()
-                                .startFromSpecificOffsets(
-                                        OFFSETS)) // test if they accepted although not needed
-                .withFormat(new TestTableFormat())
-                .withSchema(
-                        new Schema()
-                                .field(FRUIT_NAME, DataTypes.STRING())
-                                .field(COUNT, DataTypes.DECIMAL(10, 4))
-                                .field(EVENT_TIME, DataTypes.TIMESTAMP(3)))
-                .inAppendMode()
-                .toProperties();
+        final Map<String, String> map = new HashMap<>();
+        map.put("schema.0.data-type", "VARCHAR(2147483647)");
```

Comment:
nit: why not `STRING`?

Reply:
this is just the generated output from the removed code